lib/widgets/autocomplete.dart

```diff
@@ -195,24 +198,56 @@ class ComposeAutocomplete extends AutocompleteField<MentionAutocompleteQuery, Me
 
   @override
   Widget buildItem(BuildContext context, int index, MentionAutocompleteResult option) {
+    final designVariables = DesignVariables.of(context);
     Widget avatar;
     String label;
+    String? metadata;
+
     switch (option) {
```

Comment:
nit: use blank lines to separate stanzas logically:
```suggestion
    final designVariables = DesignVariables.of(context);

    Widget avatar;
    String label;
    String? metadata;
    switch (option) {
```

In particular `avatar`, `label`, and `metadata` should have their declarations remain together with the logic that initializes them.